graalpython/com.oracle.graal.python/src/com/oracle/graal/python/runtime/formatting/StringFormatter.java

```diff
@@ -15,7 +15,8 @@
 import static com.oracle.graal.python.runtime.exception.PythonErrorType.TypeError;
 import static com.oracle.graal.python.runtime.exception.PythonErrorType.ValueError;
 
-import com.oracle.graal.python.util.BiFunction;
+import java.math.BigInteger;
+import java.util.function.BiFunction;
```

Comment:
Please keep using `com.oracle.graal.python.util.BiFunction`. We use it because native image has trouble analysing the generic JDK types and thus blacklists them.

Reply:
Ah. Probably something my editor did of its own accord. Can fix.